app/views/idv/by_mail/request_letter/index.html.erb

```diff
@@ -29,9 +29,46 @@
   <%= render AlertComponent.new(message: t('idv.messages.gpo.info_alert'), class: 'margin-bottom-4') %>
   <%= render PageHeadingComponent.new.with_content(@presenter.title) %>
   <p>
-    <%= t('idv.messages.gpo.address_on_file_html') %>
     <%= t('idv.messages.gpo.timeframe_html') %>
+    <br />
+    <%= new_tab_link_to(
+          t('idv.messages.gpo.learn_more_verify_by_mail'),
+          help_center_redirect_url(
+            category: 'verify-your-identity',
+            article: 'verify-your-address-by-mail',
+            flow: :idv,
+            step: :gpo_send_letter,
+          ),
+        )
+    %>
+  </p>
+  <p>
+    <%= t('idv.messages.gpo.address_on_file') %>
   </p>
+
+  <p>
+    <%= @applicant[:address1] %>
+    <br />
+    <% if @applicant[:address2] %>
+      <%= @applicant[:address2] %>
+      <br />
+    <% end %>
+    <%= @applicant[:city] %>, <%= @applicant[:state] %> <%= @applicant[:zipcode] %>
+    <br />
+  </p>
+
+  <p>
+    <%= start_over_link_html = link_to(
+          t('idv.messages.gpo.start_over_link_text'),
+          idv_cancel_path(step: local_assigns[:step]),
```

Comment:
the path for this is `idv_confirm_start_over_path`

Reply:
Changed; thanks.